frontend-react/src/hooks/UseCreateOrganizationPublicKey.ts

```diff
@@ -0,0 +1,53 @@
+import { useCallback } from "react";
```

Comment:
I still think we should take a more intentional approach to our hooks/components rather than have a catch-all hooks/ or components/ directory, but this is fine for now.  In the future, I'd recommend we take the domain-driven approach as described [in the proposal](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/docs/proposals/0002-domain-driven-directory-structure.md).

Reply:
Agree. I can create a /hooks/PublicKeys folder or  /hooks/Organizations/PublicKeys and move. Thoughts?